src/functions/types.ts

```diff
@@ -396,3 +397,32 @@ export type RuntimeUnhandledEventContext<
     // deno-lint-ignore no-explicit-any
     body: any;
   };
+
+type DefineFunctionReturnType = ReturnType<typeof DefineFunction>;
```

Comment:
Since this is only used in one other place in this file, I would remove it. I don't think it offers much additional value, and we already suffer from having too many types and type aliases in this project as it is.

My suggestion is to just use `ReturnType<typeof DefineFunction>` directly where you need it

Reply:
This makes sense, Ive implemented this 👍